Tests/SentryCrashTests/SentryCrashInstallationTests.swift

```diff
@@ -0,0 +1,99 @@
+import SentryTestUtils
+import XCTest
+
+class SentryCrashTestInstallation: SentryCrashInstallation {
+    override init() {
+        super.init(requiredProperties: [])
+    }
+}
+
+final class SentryCrashInstallationTests: XCTestCase {
```

Comment:
Rewritten in swift, see my other comment on the old objc implementation that was removed.